### DISCUSSION TITLE
"Beskrivelse" i ugeplan

### DISCUSSION THREAD

**Discussion by Fumper1** [scaarup/aula#176] [Q&A]
Nu har jeg brugt et par dage på at lede efter den beskrivelse jeg har liggende i starten af min ugeplan, er der nogen der har haft held til at få den frem? 

jeg har prøvet om det var ugenoter eller huskelisten, men begge er null. 

![IMG_6812](https://github.com/user-attachments/assets/464655aa-02e1-4f65-8a10-7f0d42638ae2)

**Comment by PatrickFrederiksen** [comment]
Jeg har samme udfordring, at vores lærere smider en masse information på selve uge planen, og ikke på de individuelle fag.

I json objektet der kommer retur fra easyiq der ligger den data vi gerne vil have med i ugeplaner.json()["WeekPlan"]["Text"]
Men dataen der bliver smidt i ugeplans sensoren i ha kommer alt sammen fra ugeplaner.json()["Events"]

Jeg har løst det i min eget lokale ved at smidefølgende i clients.py lige før loopet der kigger på "Events" 

```
                            ### Add Week description
                            _ugep = (
                            _ugep
                            + "<br><br>"
                            + ugeplaner.json()["WeekPlan"]["Text"]
                            + "<br><br>"
                            )
```

**Comment by Fumper1** [comment]
Nu har jeg prøvet i et stykke tid at gennemskue hvor det er jeg skal ligge det, kan du give mig et snip fra din integration, så jeg kan se det tydeligere?
hvis jeg kan få det til at virke tænker jeg at det skal laves til en PR istedet.
